client/src/hooks/Input/useSpeechToTextBrowser.ts

```diff
@@ -28,7 +50,7 @@ const useSpeechToTextBrowser = () => {
 
   useEffect(() => {
     const handleKeyDown = (e: KeyboardEvent) => {
-      if (e.shiftKey && e.altKey && e.code === 'KeyL' && endpointSTT === 'browser') {
+      if (e.shiftKey && e.altKey && e.code === 'KeyL' && !useExternalSpeechToText) {
```

Comment:
is `useExternalSpeechToText` a hook or a boolean value? you should not name a variable starting with `use` in react unless it's a hook